man/man5/zfs-module-parameters.5

```diff
@@ -1660,65 +1674,69 @@ Use \fB1\fR for yes and \fB0\fR for no (default).
 .sp
 .ne 2
 .na
-\fBzfs_resilver_delay\fR (int)
+\fBzfs_resilver_min_time_ms\fR (int)
 .ad
 .RS 12n
-Number of ticks to delay prior to issuing a resilver I/O operation when
-a non-resilver or non-scrub I/O operation has occurred within the past
-\fBzfs_scan_idle\fR ticks.
+Resilvers are processed by the sync thread. While resilvering it will spend
+at least this much time working on a resilver between txg flushes.
 .sp
-Default value: \fB2\fR.
+Default value: \fB3,000\fR.
 .RE
 
 .sp
 .ne 2
 .na
-\fBzfs_resilver_min_time_ms\fR (int)
+\fBzfs_scrub_min_time_ms\fR (int)
 .ad
 .RS 12n
-Resilvers are processed by the sync thread. While resilvering it will spend
-at least this much time working on a resilver between txg flushes.
+Scrubs are processed by the sync thread. While scrubbing it will spend
+at least this much time working on a scrub between txg flushes.
 .sp
-Default value: \fB3,000\fR.
+Default value: \fB1,000\fR.
 .RE
 
 .sp
 .ne 2
 .na
-\fBzfs_scan_idle\fR (int)
+\fBzfs_scan_mem_lim_fact\fR (int)
```

Comment:
Can you also make sure all the `zfs_scan_*` entries are listed together in alphabetical order.